homeassistant/components/devolo_home_network/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "flow_title": "{product} ({name})",
+    "step": {
+      "user": {
+        "description": "[%key:common::config_flow::description::confirm_setup%]",
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add the devolo home network device with the hostname `{host_name}` to Home Assistant?",
+        "title": "Discovered devolo home network device"
+    }
```

Comment:
The indentations starting at line 14 are all off.

Reply:
fixed